guide/tutorial-4-more-about-controllers-20-minutes.md

```diff
@@ -198,11 +198,11 @@ threshold) as follows:
 
 > ```c
 > // detect obstacles
-> bool left_obstacle =
+> bool right_obstacle =
```

Comment:
These changes have to be ported to the corresponding controller in `webots` repository.
I can take care of this.